packages/protocol/migrationsConfig.js

```diff
@@ -84,11 +84,11 @@ const DefaultConfig = {
     frozen: true,
   },
   governance: {
-    queueExpiry: 4 * WEEK,
-    dequeueFrequency: WEEK,
-    concurrentProposals: 10,
-    approvalStageDuration: 3 * DAY,
-    referendumStageDuration: WEEK,
+    queueExpiry: WEEK, // Change to 4 weeks once mainnet activated
+    dequeueFrequency: DAY, // Change to 1 week once mainnet activated
```

Comment:
On second thought, given the number of governance proposals that will need to be passed, suggest speeding these up even further for RC0:

```
dequeueFrequency: MINUTE,
approvalStageDuration: 30 * minute,
referendumStageDuration: HOUR,
```